client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -72,7 +72,7 @@ export const DateFilter: FC<{ filterDefinition: DateFilterDefinition }> = ({
 
 const getDateFromUrl = (query: string, range: RangeOption | undefined) => {
   const value = typeof query !== 'object' || !range ? query : query[range];
-  return DateUtils.getDateOrNull(value);
+  return DateUtils.getNaiveDate(value);
```

Comment:
Date filter now uses Naive date, which means it will match the string in the URL as well as the displayed "value". 

I don't think this would cause any problems anywhere would it?